test/test_sqlalchemy_json.py

```diff
@@ -178,3 +179,12 @@ def test_mutable_json_list(session, author_with_list):
     session.commit()
 
     assert author_with_list.handles == ["@JohnDoe", "JohnDoe", "@mike_bianco"]
+
+
+def test_null_none(session):
+    """
+    Make sure library can handle both None and null() as JSON value
+    """
+    for value in (None, null()):
+        author = Author(name="John Doe", handles=value)
+        Article(author=author.name, content="very important", references=value)
```

Comment:
Instead of having the test fail when either of these fail, parametrize over the different null-like values:

```suggestion
@pytest.mark.parametrize("null_like", [None, null()])
def test_null_like_assignments(null_like):
    """Make sure library can handle both None and null() as JSON value."""
    author = Author(name="John Doe", handles=null_like)
    assert author.handles is None
```